spec/views/tutoring_sessions/index_spec.rb

```diff
@@ -3,15 +3,19 @@
 RSpec.describe 'tutoring_sessions/index', type: :view do
   before(:each) do
     @tutoring_session = create(:tutoring_session, location: 'library',
-                                                  start_time: '2017-03-03',
-                                                  end_time: '2017-03-03')
+                                                  hours: '3',
+                                                  start_date: '2017-01-01',
+                                                  end_date: '2017-12-13',
+                                                  session_comment: 'hi')
```

Comment:
I dont think we should be hard coding attribute values in testing...it makes thing more difficult when attributes are altered